mesos-docker/run/run.sh

```diff
@@ -522,6 +522,14 @@ function parse_args {
     fi
   fi
 
+  if [[ -n $SPARK_BINARY_PATH ]]; then
+   SPARK_FILE=${SPARK_BINARY_PATH##*/}
+  fi
+
+  if [[ -n $HADOOP_BINARY_PATH ]]; then
+   HADOOP_FILE=${HADOOP_BINARY_PATH##*/}
```

Comment:
I am not convinced.
At the very least, it requires a comment to explain what is the intend of the code. I had to double-check in the bash doc to remember the exact meaning of this variable substitution syntax.
Also it will have to be modified if we ever port the script to bash on windows.